src/main/java/algorithms/minimumSpanningTree/README.md

```diff
@@ -0,0 +1,45 @@
+# Minimum Spanning Tree Algorithms
+
+## Background
```

Comment:
could you briefly discuss the 4 properties of MST?
1. Cut a MST and you'll get 2 MSTs
2. no cycle (this and (1) is saying the same thing)
3. cycle property
4. cut property (also tells you that the minimum outgoing edge of every vertex is part of mst)
..